### PR TITLE
[FEATURE] Added nightly support for randoms

### PR DESCRIPTION
**Pull request by CapslockRSPS**
Related to issue #134.

### CODE REVIEW

File: src/main/java/org/parabot/core/parsers/randoms/PublicRandoms.java [hunk 1]
```diff
@@ -46,12 +45,14 @@ public void parse() {
...
         try {
-            File random = new File(Directories.getCachePath() + "/randoms.jar");
+            File random = new File(Directories.getCachePath() + File.separator + "randoms.jar");
             if (random.exists()) {
                 Core.verbose("Public random dependency already exists, no need to download it...");
                 return;
             }
-            String downloadLink = Configuration.GET_RANDOMS;
+
+            String downloadLink = ((Configuration.BOT_VERSION.isNightly()) ? Configuration.GET_RANDOMS + "?stable=false" : Configuration.GET_RANDOMS);
```
Comment (JKetelaar) [inline src/main/java/org/parabot/core/parsers/randoms/PublicRandoms.java]: We've a Configuration value for `?stable=false`, not sure if it's client only.
If it's client only, add it to the internal API configuration.

File: src/main/java/org/parabot/core/parsers/randoms/PublicRandoms.java [hunk 2]
```diff
@@ -46,12 +45,14 @@ public void parse() {
 
     private void download() {
         try {
-            File random = new File(Directories.getCachePath() + "/randoms.jar");
+            File random = new File(Directories.getCachePath() + File.separator + "randoms.jar");
```
Comment (JKetelaar) [inline src/main/java/org/parabot/core/parsers/randoms/PublicRandoms.java]: What if you want the nightly version, but you used to run the default version.
`randoms.jar` will still exist, though it won't download the nightly one, as it already exists.

Maybe first check if it's nightly, then append `nightly` to the file name, like `randoms-nightly.jar`.

**Comment by CapslockRSPS** [comment]
Will do!

**Comment by JKetelaar** [comment]
Don't forget to bump to internal API too

**Comment by JKetelaar** [comment]
Very good job, thanks.

Have you tested this?

**Comment by JKetelaar** [comment]
Works perfectly!

**Comment by JKetelaar** [comment]
Also solves issue #136

**Pull request merged**